Added a new file named "milestone 2.ipynb" and modified my individual analysis file

ShaohuaJoseph opened the pull request:
1. Added a file named 'milestone 2.ipynb' that should contain the information about group individual EDA. 2. changed the topic of my individual analysis file

KafkaNoNeko commented:
Hi, thanks for trying to merge everything together.

I think the main file named milestone2 should be in the submitted folder, and not in the same directory as the license.  Moreover, the EDA should come first, and then the final analysis.  I'll make another branch tomorrow night to show what I mean (kinda have something more urgent right now )

ShaohuaJoseph commented:
> Hi, thanks for trying to merge everything together.
> 
> I think the main file named milestone2 should be in the submitted folder, and not in the same directory as the license. Moreover, the EDA should come first, and then the final analysis. I'll make another branch tomorrow night to show what I mean (kinda have something more urgent right now )

Hi, Paula. Thank you for your reply. I have moved the file to the submitted directory and switched the positions of EDA and final analysis. Please take a look and see if it gets better!

KafkaNoNeko commented:
Looks better!  I made some changes to the titles

Not too sure if we can just copy our work or if we should also organise the EDA by research question though
